File: collie/src/main/java/com/snail/collie/mem/TrackMemoryInfo.java
package com.snail.collie.mem;

import android.app.ActivityManager;
import android.os.Debug;

public class TrackMemoryInfo {

    public String procName;
    public JavaMemory javaMemory;
    public NativeMemory nativeMemory;
    public SystemMemory systemMemoryInfo;
    public String display;
}

File: collie/src/main/java/com/snail/collie/mem/TrackMemoryInfo.kt
package com.snail.collie.mem

class TrackMemoryInfo {
    @JvmField
    var procName: String? = null

    @JvmField
    var appMemory: AppMemory? = null

    @JvmField
    var systemMemoryInfo: SystemMemory? = null

    @JvmField
    var display: String? = null

    @JvmField
    var activityCount = 0
}